src/helpers/themeClass.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(something) {
```

Comment:
change `something` to `theme`, because thats what youre passing in

Reply:
Will do